testutil/keeper/application.go

```diff
@@ -1,6 +1,9 @@
 package keeper
 
 import (
+	mocks "pocket/testutil/application/mocks"
+	"pocket/x/application/keeper"
+	"pocket/x/application/types"
 	"testing"
 
 	tmdb "github.com/cometbft/cometbft-db"
```

Comment:
NIT: import grouping

```suggestion
	"testing"

	tmdb "github.com/cometbft/cometbft-db"
```